tests/components/nfandroidtv/test_notify.py

```diff
@@ -0,0 +1,166 @@
+"""Test nfandroidtv notify service."""
+
+from typing import Any
+
+import httpx
+import pytest
+import respx
+
+from homeassistant.bootstrap import HomeAssistantError
+from homeassistant.components.nfandroidtv.const import DOMAIN
+from homeassistant.components.notify import DOMAIN as NOTIFY_DOMAIN
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ServiceValidationError
+
+from . import CONF_DATA
+
+from tests.common import MockConfigEntry, load_json_value_fixture
+
+
+@respx.mock
+async def test_sending_notification(hass: HomeAssistant) -> None:
+    """Test sending notification with message and title and data."""
+    respx.get("http://1.2.3.4:7676/") % 200
+    respx.post("http://1.2.3.4:7676/") % 200
```

Comment:
We would rather mock the whole library instead of mocking the HTTP calls it does